examples/ai-billing/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ai-billing",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@radix-ui/react-alert-dialog": "^1.0.5",
```

Comment:
we no longer use the radix stuff ,right?
just tremor

Reply:
I added shadCN UI back in, it's mostly used for the auth UI. Lucide isn't a drop in replacement (doesn't have dropdown menus etc). I'm just using it for components though, I remove its CSS and Tailwind modifiers so there's no clash